scripts/seed_data.py

```
@@ -0,0 +1,115 @@
+            student=student,
+            defaults={'status': enrollment_data['status']}
+        )
+    print("✅ Enrollment created successfully.")
+
+
+def run() -> None:
+    base_dir = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(base_dir, 'dummy_data.json')
+
+    try:
+        with open(file_path, 'r') as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)